### ISSUE TITLE
Extract to Journal entry Missing on Newer Journals

### ISSUE THREAD

**Issue by dale-e-o** [ironmonk88/monks-enhanced-journal#455]
All the Journals I made in v9 still have the option to Extract to journal entry
<img width="152" alt="image" src="https://user-images.githubusercontent.com/118127109/208015269-46e22abf-3835-4301-bba7-1301ff37957c.png">

All my New docs in v10 do not have the option. 
<img width="302" alt="image" src="https://user-images.githubusercontent.com/118127109/208015088-d457e73c-954c-4938-83e1-838302e3f8de.png">

Ive looked through all the options and did not find any releated
I looked at the sheet settings and I cannot select Default Sheet as journal Entry I wondered if this is the issue.

**Comment by dale-e-o** [comment]
Sorry I also asked in the foundry discord forum, they said it was a module. Monks is the only journal enhancement i have :)

**Comment by ironmonk88** [comment]
It had to be removed because you can't convert a full Adventure Book journal into a Singular Enhanced Journal page.
At least not easily.  So in the meantime, while I work on the feature, I had to disable the button.

**Comment by dale-e-o** [comment]
Ahhh its so usefull for slicing up the docs into sub entries to drop on a map , thanks

**Comment by dale-e-o** [comment]
out of interest how do I get this prioritized?

**Comment by ironmonk88** [comment]
![image](https://user-images.githubusercontent.com/75920956/210103148-7b73eeee-2d4b-437f-85de-85ae9b45d727.png)
![image](https://user-images.githubusercontent.com/75920956/210103187-625e0a71-a7d3-4687-875c-02f42ecb22a0.png)
It exists in the latest version... it just does some weird things so I'm not ready to close the ticket yet.  And it only works on a page rather than text.

**Comment by dale-e-o** [comment]
OK Ill test it a bit and give you some feedback . Much appreciated

**Comment by dale-e-o** [comment]
Yeah I can confirm it does do the whole page but not individual sections, Thanks so Much looking foward to the whole feature!

**Comment by dale-e-o** [comment]
Any Idea on a timeline on this I am reaching the point I could really use it

**Comment by bularzik** [comment]
Thanks for the report. This is fixed on our in-progress v14 migration branch (#820) — page-to-standalone-JournalEntry "Extract" is implemented via the context menu again. It'll be part of the next release once that branch lands; I don't believe there's anything further to do here in the meantime.
